content/en/tracing/profiling/_index.md

```diff
@@ -0,0 +1,91 @@
+| ----------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| CPU               | CPU load summed on all cores of the process. This means this can go above `100%` and the theoretical limit is `nCores` \* `100%`                                                    |
+| Memory Allocation | Memory allocation rate in `Bytes/s` over the course of the profile. This value can be above the maximum amount of RAM memory because it can be garbage collected during the profile |
+
+## Profile
+
+Click on one line to view a specific profile:
+
+{{< img src="tracing/profiling/profile.png" alt="A specic profile">}}
+
+The profile header contains information associated to your profile like: the service that generated it, the environment and code version associated to it. Find also in the upper right corner of each profile options to:
+
+- Download the profile
+- Switch the profile to full screen
+
+Below the profile header you can find 4 tabs:
+
+| Tab          |  Definition                                                                                                                                 |
+| ------------ | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| Profiles     | Flame graph and summary table of the profile you are looking at. You can switch between multiple profile types (`CPU`, `Memory allocation`) |
+| Analysis     | A set of heuristics that suggest potential issues or areas of improvement in your code                                                      |
+| Metrics      | Profiling metrics coming from all profiles of the same service                                                                              |
+| Runtime Info | Text dump of all the Runtime properties                                                                                                     |
+
+### Profile types
+
+In the **Profiles** tab you can see all profile types available for a given language. Depending of the language, the information collected about your profile differs. Refer to the dedicated profile types section for your language to see which one are collected:
+
+{{< partial name="profiling/profiling-languages.html" >}}
+
+## Troubleshooting
+
+Datadog profiling is in beta. It doesn't work with some configurations or environments:
+
+- Datadog profiling does not work with serverless.
+- Datadog profiling does not work with complex proxy configuration. Profiles are sent directly from your application to Datadog.
+
+In case you have done all the necessary configuration steps and do not see profiles in the [profile search page](#search-profiles), please turn on [debug mode][4] and open a support ticket with debug files and following information:
```

Comment:
```suggestion
In case you have done all the necessary configuration steps and do not see profiles in the [profile search page](#search-profiles), turn on [debug mode][4] and open a support ticket with debug files and the following information:
```